src/controllers/user.js

```diff
@@ -121,6 +121,50 @@ const UserController = (userModel, authService, googleAPIService) => {
     });
   });
 
+  router.post('/weekend', async (req, res)=>{
+    if (!req.headers) return res.status(400).json({
+        data: null,
+        message: "Malformed Request"
+    });
+
+    if (!req.body) return res.status(400).json({
+      data: null,
+      message: "Malformed Request",
+    });
+    body = req.body;
+
+    const weekend_setting = body.weekend_setting
+    if(weekend_setting === undefined){
+      return res.status(400).json({
+        data:null,
+        message:"Malformed Request"
+      });
+    }
+
+    const [id, err1] = await authService.getLoggedInUserID(req.headers);
+
+    if(id === undefined){
+      return res.status(400).json({
+        data:null,
+        message:"Malformed Request: "+err1,
+      });
+    }
```

Comment:
Just for consistency, can we add spaces after `if` and the condition? So 

```
if (weekend_setting === undefined) {
      return res.status(400).json({
        data:null,
        message:"Malformed Request"
      });
    }
```
Sorry I know this is nitpicky but this is what a review will look like at a company as well

Reply:
Looks like there are a few places to make that change in this file